<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="py_QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons/icons.qrc">
    <normaloff>:/icons/tadek-icon.png</normaloff>:/icons/tadek-icon.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="locale">
     <locale language="English" country="UnitedKingdom"/>
    </property>
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionDevices"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBarSwitch">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>4</width>
     <height>13</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBarMain">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>136</width>
     <height>57</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <addaction name="actionDevices"/>
   <addaction name="actionSettings"/>
  </widget>
  <widget class="QToolBar" name="toolBarView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>4</width>
     <height>13</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionDevices">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/icons/phone.png</normaloff>:/icons/phone.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Devices</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/icons/configure.png</normaloff>:/icons/configure.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>py_QMainWindow</class>
   <extends>QWidget</extends>
   <header>py_qmainwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
